components/Header/index.js

```diff
@@ -101,6 +101,14 @@ export default function Header(props) {
 
         {/* CTA Wrapper */}
         <div className="hidden items-center justify-end md:flex md:flex-1 lg:w-0">
+          <a href="https://forms.gle/953YD2a4fbbsvp9V6" target="_blank">
+            <button className="relative inline-flex items-center justify-center p-0.5 mb-1.5 mr-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-blue-700 to-blue-300 hover:text-white dark:text-white">
```

Comment:
My personal vote would be to avoid adding more colors - the blue seems to clash a bit. We have a general theme of white, gray, black, and purple hues. 

What about something like this where we have a lighter purple and a darker purple button?

```suggestion
            <button className="relative inline-flex items-center justify-center p-0.5 mb-1.5 mr-2 overflow-hidden text-sm font-medium text-gray-900 rounded-lg group bg-gradient-to-br from-medpurple to-lightpurple group-hover:from-medpurple group-hover:to-lightpurple hover:text-white dark:text-white">
```

Maybe we explore having the `Register Now!` button have a colored purple background to start and when someone hovers over it, the text goes to purple and the background to white? Almost do the opposite of what the `Join Discord Server` button does?

Reply:
alright, here's what that looks like
![image](https://github.com/Minnesota-Computer-Club/MCC-Website-v2/assets/29520825/e6a82544-9f1d-479e-9759-049a4d1b801b)

when hovering over the join discord button and register now button it looks like they hover at the same time, but that's fine I think